Templates/Android/generated/BaseEntityCollectionRequestBuilder.java.tt

```diff
@@ -52,6 +52,15 @@ if (currentTypeProjection != null) {
         return new <#=TypeCollectionRequestBuilder(method)#>(getRequestUrlWithAdditionalSegment("<#=MethodFullName(method)#>"), getClient(), null<#=MethodParametersValues(method)#>);
     }
 <#
+	// Add a method to support delta query by string (opaque deltaLink)
```

Comment:
I think this is fine. I understand this is used to generate a request builder for OData functions and actions bound to collections. Instead of the method name, we are appending the deltalink to the URL.  I might understand this.